msal/src/telemetry/java/com/microsoft/identity/client/Event.java

```diff
@@ -74,13 +83,44 @@ String getEventName() {
     }
 
     /**
+     * Gets the Event startTime.
+     *
+     * @return the startTime to get.
+     */
+    Long getStartTime() {
+        return Long.valueOf(getProperty(EventProperty.START_TIME));
+    }
+
+    /**
+     * Gets the Event stopTime.
+     *
+     * @return the stopTime to get.
+     */
+    Long getStopTime() {
+        return Long.valueOf(getProperty(EventProperty.STOP_TIME));
+    }
+
+    /**
+     * Gets the Event elapsedTime.
+     *
+     * @return the elapsedTime to get.
+     */
+    Long getElapsedTime() {
+        return Long.valueOf(getProperty(EventProperty.ELAPSED_TIME));
+    }
+
+    /**
      * Builder object used for Events.
      *
      * @param <T> generic type parameter for Builder subtypes.
      */
     abstract static class Builder<T extends Builder> {
 
         private final String mEventName;
+        private Long mEventStartTime;
```

Comment:
I used a `Long` object type because this value can be `null` (in the case of an `OrphanedEvent` or if an `Event` is created but never started) - it would represent a bug in the flow of the application, but it's something that is 'planned for' with the `OrphanedEvent` code